packages/voting/graphql/resolvers/_mutations/createElection.js

```diff
@@ -18,11 +18,13 @@ module.exports = async (_, { electionInput }, context) => {
       throw new Error(t('api/election/exists'))
     }
 
+    /* TODO: would be nice to have the possibility to set discussionId via electionInput
+    because if discussion is created via publikator we cannot set id there */
```

Comment:
Not fond of `TODO` notes; either do it or remove it. (Hints on code improvement could still remain, just without `TODO` flag.)